js/components/generate/generate.js

```diff
@@ -224,6 +224,7 @@ class GenerateForm extends React.Component {
     <Graph
       ref={this.graph}
       start_blank={true}
+      graphs={[]}
```

Comment:
I think it's better to actually add this as a default prop to the `Graph` component. See the `defaultProps` class attribute.